app/scripts/lib/storage.js

```diff
@@ -17,6 +17,16 @@ define(function (require, exports, module) {
     return NAMESPACE + '.' + key;
   }
 
+  function normalizeError(error, type) {
+    error.context = 'storage';
+    error.namespace = type;
+    // Firefox localStorage errors contain a `name` field. Report that
+    // name if available, otherwise return the whole message.
+    error.errno = error.name || error.message;
```

Comment:
Here is a decoded example of what is reported to Sentry:

<img width="1010" alt="screen shot 2016-01-26 at 15 24 54" src="https://cloud.githubusercontent.com/assets/848085/12584781/01186d5a-c441-11e5-87a8-cfd0d6c9c81c.png">


And the error reported to internal metrics:

<img width="553" alt="screen shot 2016-01-26 at 15 23 15" src="https://cloud.githubusercontent.com/assets/848085/12584796/17d6b3a8-c441-11e5-89cf-ea20385ed236.png">